tslint.json

```diff
@@ -0,0 +1,9 @@
+{
+	"defaultSeverity": "error" ,
+	"extends": [
+		"tslint-config-airbnb"
+	],
+	"rules": {
+		"trailing-comma": [ true ]
+	}
+}
```

Comment:
agrega el fin de línea (EOF que significa end of line) es una línea vacía para facilitar la lectura del archivo

Reply:
Done!